doc/interfaces/denoise.rst

```diff
@@ -0,0 +1,77 @@
+=========
+Denoising
+=========
+
+This tutorial walks through the steps to denoise diffusion-weighted MR images using DIPY.
+Multiple denoising methods are available in DIPY.
+
+First you need to download the desired data.
```

Comment:
Maybe something like:

You can try these methods using your own data; we will be using the data in DIPY. In order to know how to get the DIPY data, you can read [link_to_data_fetch_cli_rst](link_to_data_fetch_cli_rst).

Reply:
Addressed in 7ba392f

Reply:
It is recommended the link to be a relative one, similar to the suggested for [this other comment](https://github.com/dipy/dipy/pull/2248#discussion_r500684909).

So in this case it would be: `fetch the DIPY data <data_fetch.rst>`_ if I am not mistaken.

Reply:
<data_fetch.rst>_ did not work

Reply:
please, check the sphinx doc when it does not work 😉 

```
:ref:`fetch the DIPY data<data_fetch.rst>`
```

Reply:
Addressed in 404d997